test/shapes_test.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace test\shapes;
+
+require("../lib/shapes.php");
+
+use PHPUnit\Framework\TestCase;
+use shapes;
+
+class shapes_testcase extends TestCase {
+
+    public function test_square_build(){
+        $this->assertEquals("###\n###\n###\n", shapes\build_square(3));
```

Comment:
Can you write 2 more assertions for 2 and 5 as the square size? Would make sure for you that it produces what you expect also with other numbers.